openapi.yml

```diff
@@ -48,24 +48,26 @@ components:
     DataLoadJob:
       description: Information used to schedule a data load job in the pipeline
       type: object
-      additionalProperties: false
+      additionalProperties: true
```

Comment:
Should this be `additional_properties`?

Reply:
No, I don't think so as this is a configuration option to allow additional properties that aren't specifically defined/validated here, not a field we'll be including in the data.